.scrutinizer.yml

```diff
@@ -0,0 +1,8 @@
+build:
+    tests:
+        override:
+            -
+                command: 'phpdbg -qrr vendor/bin/phpunit --coverage-clover clover.xml'
```

Comment:
actually this command works much faster than scrunitizer (travis spends most of a time on humbug)